aquarius/events/processors.py

```diff
@@ -634,24 +634,32 @@ def restore_ddo(self):
     def process(self):
         self.did = make_did(self.event.address, self._chain_id)
         # check if assets exists. if not, bail out
-        exists = self._es_instance.exists(self.did)
-        if not exists:
+        ddo = self._es_instance.read(self.did)
+        soft_delete_stats = [
```

Comment:
I would suggest defining the soft delete states (btw, that's a typo there, stats instead of states) somewhere in the same file or class as MetadataStates itself. This definition is much like a single constant, so it should be suggestive of that.